android/build.gradle

```diff
@@ -17,7 +17,7 @@ android {
     buildToolsVersion '29.0.2'
 
     defaultConfig {
-        minSdkVersion 19
+        minSdkVersion 21
```

Comment:
[Curiosity] Do we have a reason to upgrade `minSdkVersion`?

Reply:
Android 21 (lollipop) was released in 2014. Hardly anyone is still using this. Targeting a lower android version had other warnings in the code that had to be fixed.

Reply:
Thanks